Use the EvmDatabase method for evm-failover-monitor ExecStart

carbonin opened the pull request:
Depends on https://github.com/ManageIQ/manageiq/pull/17837

The bin file was removed from the gem in https://github.com/ManageIQ/manageiq-postgres_ha_admin/pull/10

carbonin committed:
Remove manageiq-postgres_ha_admin from the appliance dependencies

We use the gem from manageiq's core repo now, so we will move
the dependency there.

https://bugzilla.redhat.com/show_bug.cgi?id=1391095

carbonin committed:
Use the rails runner to start the failover monitor

https://bugzilla.redhat.com/show_bug.cgi?id=1391095

jrafanie commented on COPY/usr/lib/systemd/system/evm-failover-monitor.service:
```diff
@@ -3,7 +3,7 @@ Description=PostgreSQL Failover Monitor for ManageIQ
 
 [Service]
-ExecStart=/bin/sh -l -c "bundle exec postgres_ha_admin"
+ExecStart=/bin/sh -l -c "bundle exec rake evm:db:run_failover_monitor"
```
Perhaps you can use bundle exec rails runner method name

carbonin committed:
Use a script for running the rails runner command

Without this, the service fails and prints the 'rails new'
help to the journal logs

jrafanie commented:
@carbonin dependency was merged... can you unwip it?  Is there anything else needed here?
